dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties

```diff
@@ -18,6 +18,9 @@ blob.export.localFile.directory=file://var/blobExporting
 # Mandatory if you choose LinShare, url to connect to LinShare service
 # blob.export.linshare.url=http://linshare:8080
 
-# Mandatory if you choose LinShare, access token to connect to LinShare service. It will be formalized to `Bearer` + a space + access token
-# So, no need to pass (`Bearer` + <space>) prefix
-# blob.export.linshare.token=LinShare-Access-Token-In-String
\ No newline at end of file
+# ======================================= LinShare Configuration BasicAuthentication ===================================
+# Authentication is mandatory if you choose LinShare, TechnicalAccount is need to connect to LinShare specific service.
+# For Example: It will be formalized to 'Authorization: Basic {Credential of UUID/password}'
+
+# blob.export.linshare.user.name=Technical_Account_UUID
```

Comment:
It is no longer a name but a UUID, then this property should be named `blob.export.linshare.tachnicalAccount.uuid`

Reply:
I agree.